### ISSUE TITLE
Pinning packages in requirements.txt

### ISSUE THREAD

**Issue by keelezibel** [assafelovic/gpt-researcher#336]
There is dependency conflict in `tiktoken` package between `gpt-researcher` and `langchain-openai`. 

```
langchain-openai 0.0.1 depends on tiktoken<0.6.0 and >=0.5.2
tavily-python 0.2.8 depends on tiktoken==0.5.1
```

Also, is it possible to create separate packages for different retrievers? e.g. (`gpt-researcher[all]`, `gpt-researcher[tavily]` etc.)

**Comment by gregdrizz** [comment]
we'll fix the dependency issues soon, i'll keep you posted.

**Comment by assafelovic** [comment]
fixed here #338
